client/containers/DashboardOverview/overviewRows/PubOverviewRow.tsx

```diff
@@ -19,13 +24,46 @@ type Props = {
 	className?: string;
 	pub: Pub;
 	inCollection?: boolean;
+	isSubmission?: boolean;
+};
+
+const labelPairs = (iconLabelPairs: IconLabelPair[]) => {
+	return (
+		<div className="summary-icons">
+			{iconLabelPairs.map((iconLabelPair, index) => {
+				const {
+					icon,
+					label,
+					iconSize: iconLabelPairIconSize = 12,
+					intent = 'none',
+				} = iconLabelPair;
+				const iconElement =
+					typeof icon === 'string' ? (
+						<Icon icon={icon} iconSize={iconLabelPairIconSize} intent={intent} />
+					) : (
+						icon
+					);
+				return (
+					<div
+						className="summary-icon-pair"
+						// eslint-disable-next-line react/no-array-index-key
+						key={index}
+					>
+						{iconElement}
+						{label}
+					</div>
+				);
+			})}
+		</div>
+	);
 };
 
 const PubOverviewRow = (props: Props) => {
 	const {
 		pub,
 		className,
 		inCollection,
+		isSubmission,
```

Comment:
Gonna nitpick here and say that this should be `hasSubmission` to be consistent with how we model `Submissions` and `Pubs`.